src/toil/batchSystems/abstractGridEngineBatchSystem.py

```diff
@@ -107,10 +107,10 @@ def createJobs(self, newJob: Any) -> bool:
             while len(self.waitingJobs) > 0 and \
                     len(self.runningJobs) < int(self.boss.config.maxLocalJobs):
                 activity = True
-                jobID, cpu, memory, command, jobName = self.waitingJobs.pop(0)
+                jobID, cpu, memory, command, jobName, environment = self.waitingJobs.pop(0)
```

Comment:
Hmm. It looks like [`htcondor`](https://github.com/DataBiosphere/toil/blob/releases/5.4.x/src/toil/batchSystems/htcondor.py#L316) is including the number of disks as well, so it might be worth it to keep this generic for subclasses?